### PR TITLE
Remove pymanopt

### PR DESCRIPTION
**Pull request by ferrine**
Fix an issue in #73 and #74

### CODE REVIEW

File: tests/test_manifold.py [hunk 1]
```diff
@@ -73,117 +28,140 @@ def manifold(request, retraction_order):
...
+        elif issubclass(Manifold, geoopt.manifolds.PoincareBall):
+            ex = torch.randn(*shape) / 3
+            ev = torch.randn(*shape) / 3
+            x = torch.tanh(torch.norm(ex)) * ex / torch.norm(ex)
+            ex = x.clone()
+            v = ev.clone()
+
+            manifold = Manifold()
+        elif issubclass(Manifold, geoopt.SphereSubspaceComplementIntersection):
+            compliment = torch.rand(shape[-1], 1)
```
Comment (newkozlukov) [inline tests/test_manifold.py]: it's "complement"

**Comment by newkozlukov** [comment]
Tldr though: no need for `elif`s, no need to spread test cases along multiple dicts and logic blocks in different parts of the file; and we'll need to write manifold specific tests at some point; pretty sure there's lot of corner cases we've never analyzed
